testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -40,13 +40,16 @@
 from MDAnalysisTests.util import block_import
 
 
+
```

Comment:
I was playing with the code so forgot to remove extra lines.Apology for this carelessness will not
happen again.